vllm/entrypoints/openai/serving_chat.py

```diff
@@ -522,11 +524,14 @@ def _get_top_logprobs(
             self, logprobs: Dict[int, Logprob], top_logprobs: Optional[int],
             tokenizer: PreTrainedTokenizer) -> List[ChatCompletionLogProb]:
         return [
-            ChatCompletionLogProb(
-                token=(token := self._get_decoded_token(p[1], p[0],
-                                                        tokenizer)),
-                logprob=max(p[1].logprob, -9999.0),
-                bytes=list(token.encode("utf-8", errors="replace")))
+            ChatCompletionLogProb(token=(token := self._get_decoded_token(
+                p[1],
+                p[0],
+                tokenizer,
+                return_as_token_id=self.return_tokens_as_token_ids)),
+                                  logprob=max(p[1].logprob, -9999.0),
+                                  bytes=list(
+                                      token.encode("utf-8", errors="replace")))
```

Comment:
Maybe format it like this to be easier to read?

```suggestion
            ChatCompletionLogProb(
                token=(token := self._get_decoded_token(
                    p[1], p[0], tokenizer, return_as_token_id=self.return_tokens_as_token_ids)),
                logprob=max(p[1].logprob, -9999.0),
                bytes=list(token.encode("utf-8", errors="replace")),
            )
```

Reply:
I agree, but `yapf` insists on this formatting

Reply:
Oh well...